components/TextField/textField.module.css

```diff
@@ -31,6 +31,10 @@
   outline-color: palevioletred;
 }
 
+.textField_errorMessage {
```

Comment:
The good thing about CSS Modules is that your class names can be short, so it's quicker to write/read them, for example:

- `.textField_container` -> `.container`  
- `.textField_input` -> `.input`  
- `.textField_error` -> `.input_error` (it means error is a variant of input, similar to [BEM methologody]())
- `.textField_errorMessage` -> `.errorMsg`